packages/flutter/lib/src/animation/tween.dart

```diff
@@ -246,7 +246,7 @@ class Tween<T extends Object?> extends Animatable<T> {
 
   /// Returns the value this variable has at the given animation clock value.
   ///
-  /// The default implementation of this method uses the [+], [-], and [*]
+  /// The default implementation of this method uses the `+`, `-`, and `*`
```

Comment:
How would I refer to operator+? Just use `[T.operator+]`? Or is it just not possible in dartdoc, because it doesn't know what T is?

Reply:
Yes, we don't currently use our knowledge of what T is.  We could add this feature later if we decide we really like it -- dartdoc does have enough data to figure this out.  Though in this case, since T is defined as "extending Object?" it won't have an operator+.